avm/res/app/job/main.bicep

```
@@ -30,19 +69,19 @@ param roleAssignments roleAssignmentType
 param enableTelemetry bool = true
 
 @description('Required. List of container definitions for the Container App.')
-param containers array
+param containers containerType
 
 @description('Optional. List of specialized containers that run before app containers.')
-param initContainersTemplate array?
+param initContainersTemplate initContainerType?
 
 @description('Optional. Required if TriggerType is Event. Configuration of an event driven job.')
-param eventTriggerConfig object?
+param eventTriggerConfig jobConfigurationEventTriggerConfigType?
```

Hey @ReneHezser any reason for deviating the UDT name? No side effect in terms of functionality, but unless needed I'd suggest to stay aligned to a `<parameterName>Type` convention, e.g. in this case
```suggestion
param eventTriggerConfig eventTriggerConfigType?
```

---

Same comment for all types not following the above

---

I was following the naming from the Bicep documentation page for the Container App Job. But it looked weird, you are right. Changing it as to your suggestion.